tests/robustness/snapstore/snapstore.go

```diff
@@ -0,0 +1,22 @@
+// Package snapstore describes entities that can accept
+// arbitrary metadata and flush it to a persistent repository.
+package snapstore
+
+import "github.com/kopia/kopia/tests/robustness/snapif"
+
+// Storer describes the ability to store and retrieve
+// a buffer of metadata, indexed by a string key.
+type Storer interface {
```

Comment:
Given the number of methods in the interface, it may no longer make sense to call it `Storer`.
If the package name is changed, then changing the interface name to `Store` works. WDYT?

Reply:
Also, for the particular use case here, it may not be necessary to separate the interface from the implementation. Even if one is testing a completely different tool, you could always use and require kopia for the metadata store.

Reply:
Renamed `Store` and kept the interface for now. Don't think it's worth putting in more time to make it more operationally restrictive at this point. The interface could help with boiled down unit tests that can test the Checker functionality independent of needing a kopia binary and creating a kopia repo.